tests/vppinputdecode.cpp

```diff
@@ -51,6 +51,13 @@ bool VppInputDecode::config(NativeDisplay& nativeDisplay)
     return status == DECODE_SUCCESS;
 }
 
+void VppInputDecode::getTimeStamp(VideoDecodeBuffer& inputBuffer){
```

Comment:
This will break normal decode logical, we should only set this we need measure latency.